shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/context/CommonBeans.java

```diff
@@ -180,6 +184,18 @@ MessageReaderAdapter messageReaderAdapter(
     return new MessageReaderAdapter(filteringMessageReader, sourceCatalog.getHiveMetastoreUris());
   }
 
+  private Map<String, TableReplication> transformTableReplicationsToMap(TableReplications tableReplications) {
```

Comment:
Maybe make an object to encapsulate this. `ShuntingYardTableReplications`? It can have some public look up methods that the ContextFactory can use.

Reply:
done.